Gemfile

```diff
@@ -10,7 +10,7 @@ gem 'metadata-json-lint'
 gem 'puppetlabs_spec_helper', '>= 1.1.1'
 gem 'facter', '>= 1.7.0'
 gem 'rspec-puppet'
-gem 'puppet-lint', :git => 'https://github.com/rodjek/puppet-lint.git'
```

Comment:
Leave this here and I'll address the style in another PR so it matches the other modules and uses puppet-lint v2